redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/provision/event/allocate/AllocateDatabaseRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.sequenceiq.redbeams.flow.redbeams.provision.event.allocate;
+
+import com.sequenceiq.redbeams.flow.stack.RedbeamsEvent;
+
+/**
+ * A request for allocating a database.
+ */
+public class AllocateDatabaseRequest extends RedbeamsEvent {
```

Comment:
Nit: "AllocateDatabaseRequested" sounds more like an event name.

Also, I notice it isn't used in RedbeamsProvisionEvent. Is this event redundant somehow?

Reply:
I'm going to keep it at "Request" to match the other services. This is used elsewhere -- it isn't technically linked into the provision events, but used by a handler later.